package.ps1

```diff
@@ -251,3 +251,5 @@ if ($Env:CI) {
 }
 
 echo "Rotki $SETUP_VERSION was build successfully"
+deactivate
+cd ../../..
```

Comment:
Could you add a check here to only run this if the environment is not `CI`

The CI build doesn't run in a virtualenv and deactivate won't exist so I am pretty sure this will actually break the build.
Also, this is more of a nitpick there should be an environment variable `$PROJECT_DIR` somewhere in the script. You  could do cd `$PROJECT_DIR` instead of `cd ../../..`